src/utils/streamingPayments.ts

```diff
@@ -81,7 +81,7 @@ export const getStreamingPaymentAmountsLeft = (
     amountAvailableToClaim = amountAvailableSinceStart.sub(amountClaimedToDate);
 
     amountAvailableToClaim = amountAvailableToClaim.lt(0)
-      ? BigNumber.from(0)
+      ? amountAvailableToClaim.mul(-1)
```

Comment:
If the amount is less than 0, showing 0 seems more correct than negating the amount. Are you handling a specific edge case here?

Reply:
I have to check this because maybe this is something that I've added before your fixes

Reply:
I wasn't needed anymore, so I reverted this change